b2sdk/transfer/emerge/planner/planner.py

```diff
@@ -627,8 +627,10 @@ def get_plan_id(self):
 
 class StreamingEmergePlan(BaseEmergePlan):
     def __init__(self, emerge_parts_iterator):
-        emerge_parts, self._is_large_file = self._peek_for_large_file(emerge_parts_iterator)
-        super(StreamingEmergePlan, self).__init__(emerge_parts)
+        emerge_parts_iterator, self._is_large_file = self._peek_for_large_file(
+            emerge_parts_iterator
+        )
+        super(StreamingEmergePlan, self).__init__(emerge_parts_iterator)
```

Comment:
```suggestion
        super().__init__(emerge_parts_iterator)
```

Reply:
die already, Python2! Also - when you touch a public method without type hints, please add type hints